












---
title: "The RNA and Protein Determinates Governing the HNRNPH1-Dependent Inclusion or Exclusion of EWSR1 Exon 8"
tags: ["Natasha Caplen"]
weight: 12
author: Katherine Hall
length: Short           
---

Katherine Hall, Tayvia Brownmiller, Tam Vo, Mayank Tandon, Alexei Lobanov, Katelyn Ludwig, Tamara L. Jones, Carla Neckles, Natasha J. Caplen

- Submitter: Katherine Hall (Student or postbac)
- email: hallkl@nih.gov
                  
About 85% of Ewing sarcomas (EWS) harbor a translocation that fuses a portion of the EWSR1 gene (promoter through either part of introns 7 or 8) to the 3' end of the FLI1 gene (parts of either introns 5 or 6 through exon 9 and the 3' untranslated region). This fusion oncogene, termed EWS-FLI1, expresses the oncoprotein EWS-FLI1 that promotes EWS tumorigenesis. In about a third of EWS-FLI1-driven tumors, the breakpoint in the EWSR1 gene occurs within intron 8. In these tumors, the expression of the protein-encoding fusion transcript requires exclusion of EWSR1-exon 8. We have shown HNRNPH1 mediates the splicing event that excludes EWSR1-exon 8 from the EWS-FLI1 pre-mRNA expressed in this subset of EWS. There are over twenty reported EWSR1 transcript variants, several of which are protein-encoding. The dominant EWSR1 isoform – EWSR1-207 – has 17 exons, including exon 8, but two rare variants involve splicing events centered on EWSR1-exon 8, one that skips EWSR1-exon 8 and 9 (EWSR1-222) and one that makes use of internal splice sites with Exon 8 and Exon 11 (EWSR1-201). Our previous studies have demonstrated HNRNPH1 can bind G-rich sequences at the 3' end of EWSR1-exon 8 flanking the internal splice site and that in vitro, in the presence of monovalent cations, these sequences can form stable G-quadruplex structures. We are investigating the RNA and protein determinates of EWSR1-exon 8 exclusion during EWS-FLI1 mRNA biogenesis, its retention in the context of the dominant full-length EWSR1 transcript expressed from the unaffected EWSR1 allele, and whether HNRNPH1 functions in regulating the alternative splicing of EWSR1.

Our studies are employing complementary molecular, biochemical, and biophysical methods. For example, we are using long-read RNA-seq to assess an individual gene’s transcript architecture following the silencing of HNRNPH1. Our preliminary analysis using samples generated from TC-32 cells (an EWS cell line that harbors an EWSR1-intron 8 breakpoint) indicates that the only effect of HNRNPH1 silencing on EWS-FLI1 processing is the inclusion of EWSR1-exon 8. We are currently assessing the same data to determine whether the depletion of HNRNPH1 alters the splicing of the EWSR1 pre-mRNA. Our biochemical studies include analysis of HNRNPH1's binding of the G-rich regions located at the 3' end of EWSR1-exon 8. Using RNA oligomers corresponding to one of these G-rich regions and lysates harvested from TC-32 cells and an HNRNPH1 antibody, we observed that HNRNPH1 binds the wild-type sequence but not mutant sequences. Using RNA-IP and mass spectrometry, we have also identified candidate proteins, including several RNA helicases, that may contribute to the inclusion or exclusion of EWSR1-exon 8, and we are currently validating these results using a variety of biochemical and biophysical assays. Collectively, our studies will elucidate the determinants of the alternative splicing of EWSR1-exon 8 and enhance our understanding of the function of HNRNPH1 in RNA processing.

Acknowledgment: We thank the CCR-Sequencing Facility PacBio and Bioinformatics Groups, and the NHLBI Biophysics Core Facility for their assistance with this project.



















