




---
title: "Increased Leukemic Epigenetic Plasticity and Pre-Treatment T-Cell Activation Predict Resistance to CD19 CAR T Cell Therapy in Pediatric Acute Lymphoblastic Leukemia"
tags: ["Javed Khan"]
weight: 4
author: Katherine E. Masih
length: Short           
---

Katherine E. Masih, David Milewski, Abdalla Abdelmaksoud, Hsien-Chao Chou, Berkley E. Gryder, Rebecca Gardner, Ashley L. Wilson, Serifat Adebola,  Benjamin Z. Stanton, Young K. Song, Chaoyu Wang, Xinyu Wen, Zachary Rae, Allison Ruchinskas, Adam Cheuk, Grégoire Altan-Bonnet, Michael Kelly, Jun S. Wei, Michael C. Jensen, Rimas J. Orentas, and Javed Khan

- Submitter: Katherine E. Masih (Student or postbac)
- email: katherine.masih@nih.gov
                  
Background
        Acute lymphoblastic leukemia (ALL) is the most common childhood cancer with a peak incidence between 3-5 years of age. Despite the improved survival rate of 90% for newly diagnosed children with ALL, the outcome for patients who relapse is poor with a less than 30% survival. CD19 CAR T-cell therapy has shown remarkable response rates between 80-90% in relapsed/refractory disease, which has resulted in FDA approval for two products against leukemia/lymphoma. However, 30 - 70% of initial responders will eventually relapse, and little is known about antigen-independent factors that predict initial resistance to CD19 CAR T-cell therapy. We hypothesized that leukemias that are resistant to CD19 CAR T-cell therapy are distinct from sensitive leukemias and that these differences can be detected prior to therapy. 

Methods
To interrogate differences in the genomes, transcriptomes, intratumoral heterogeneity, and epigenetic landscapes between resistant and sensitive leukemias, we obtained pre-treatment bone marrow aspirates (BMAs) from patients enrolled in a clinical trial at Seattle Children’s Hospital (PLAT-02). Samples were categorized based on patient response and included 7 resistant (2 with matched post-treatment) and 7 sensitive leukemias as matched controls. We performed multi-omic analyses, including whole exome sequencing, bulk RNA-seq, array-based methylation, single cell RNA-seq, CyTOF, ATAC-seq, and H3K27ac ChIP-seq. Additionally, we established CD19 CAR T resistant and sensitive patient derived xenografts (PDXs) of these leukemias, for in vivo modelling of these diseases.

Results
We identified mutations and fusions in known epigenetic modifiers, including CREBBP and EP300, in all resistant leukemias. ATAC-seq revealed relatively more accessible chromatin in resistant patients in comparison to sensitive (16,082 vs. 6,070 peaks). The combination of altered epigenetic modifiers and increased chromatin accessibility is suggestive of epigenetic plasticity.  Motif analysis of these differentially accessible peaks showed an enrichment for AP-1 binding proteins, particularly JUN and its partners, which provides a mechanism to avoid caspase-induced apoptosis, the downstream effect of cytotoxic T-cell killing. Methylation array and expression data indicate promoter hypermethylation and lowered expression of JUN pathway genes in sensitive leukemias.  CyTOF identified a subpopulation with characteristics of both myeloid and lymphoid lineages in one sample that may represent a resistant population of cells selected for under CAR T-cell pressure.   Analysis of the pretreatment marrow microenvironment indicates that patients with sensitive leukemias have activated T-cell populations, suggestive of more robust T-cells capable of rejecting the leukemias.

Conclusions
This study establishes one of the most integrative and comprehensive multi-omic profiling approaches for patient samples. Our results indicate the association of epigenetic plasticity, JUN pathway upregulation, and decreased T-cell activation with resistance to CD19 CAR T-cell therapy. We are currently validating these results by modelling our cohort’s therapy resistance in vivo. 




























