














---
title: "EWS-ETS Fusion Oncoproteins Suppress the Expression of Transcriptional Regulators of Cell Lineage or Differentiation"
tags: ["Natasha Caplen"]
weight: 14
author: Vernon Justice Ebegboni
length: Short           
---

Vernon Ebegboni, Tamara L. Jones, Soumya Sundara Rajan, Carla Neckles, Allison Cross, Natasha J. Caplen

- Submitter: Vernon Justice Ebegboni (Postdoc or fellow)
- email: vernon.ebegboni@nih.gov
                  
In most solid malignancies, an essential determinant of the plasticity required for metastasis is the genetic heterogeneity of the primary tumor that reflects an increasing mutational burden over time. However, in Ewing sarcoma (EWS), an aggressive bone and soft tissue sarcoma affecting children and young adults, there is a single oncogenic mutation that drives tumor development and must also regulate metastasis. The primary oncogenic event in most cases of EWS involves the EWSR1 locus and one of two members of the ETS family of transcription factors, FLI1 or ERG. The resulting fusion oncogenes (EWS-FLI1 or EWS-ERG) express aberrant transcription factors, collectively referred to as EWS-ETS. The EWS-ETS transcription factors can function as activators and repressors of gene expression. Most studies of the EWS-ETS oncoproteins have focused on their activation of genes that favor cell proliferation, but recent studies have highlighted that repressed gene targets may be responsible for the metastatic phenotype of EWS cells. Specifically, Franzetti and coworkers1 presented evidence that the heterogeneous expression of EWS-ETS within a tumor results in a subset of tumor cells that lower levels of EWS-ETS and thus elevated levels of expression of genes linked to phenotypes required for metastasis such as cell movement. Here, we aim to identify gene regulatory networks that elevated levels of EWS-ETS suppresses but which become activated when EWS-ETS levels are lower, with a focus on transcription factors involved in defining cell lineage or differentiation.
To identify genes repressed by EWS-ETS fusion oncoproteins, we first depleted EWS-FLI1 via RNAi in two EWS cell lines, TC-32 and TC-71, and performed RNA-seq analysis of the EWS-FLI1-silenced cells versus control cells (siNeg-transfected). Using the criteria of a ±1.5-fold change, FDR of <0.05, and a minimum median of four transcripts per million (TPM) in EWS-FLI1-silenced cells, we categorized 1325 genes as exhibiting an increased expression in both cell lines. To identify transcription factors within this gene set, we focused on a curated set of proteins with bona fide DNA binding activity2 and identified 114 that showed significant increases in expression following depletion of EWS-FLI1. Using published epigenetic marks for EWS cells3 and relative expression of these genes in EWS tumors and cell lines4, we selected 23 genes for further analysis. First, we assessed the regulation of these 23 genes by EWS-ETS in two further EWS cell lines, silencing EWS-FLI1 in SK-N-MC cells and EWS-ERG in ES5838. Using qPCR, we observed a significant increase in the expression of at least half of the 23 transcription factor genes in both cell lines. Based on these findings, we are currently generating a panel of TC32 EWS cell lines in which we have overexpressed each of our first genes of interest, specifically, ETS1, SNAI2, or JUNB. Following confirmation of the ectopic-expression of each gene, we subjected the modified cells to RNA-seq and compared their expression profiles to those of control cells. A preliminary analysis of TC-32 cell lines over-expressing ETS1 using the criteria of ±1.5-fold change and adjusted p-value of <0.05, we categorized 130 genes that increased in expression following the overexpression of ETS1 in TC32 cell line, of which two-thirds are genes previously shown to exhibit increased expression following silencing of EWS-FLI1, including genes involved in cell adhesion and immunomodulation. Future studies will further validate the expression and epigenetic profiles of the TC-32-ETS1 cells and EWS cells expressing other transcription factor genes of interest.

References: (1) Franzetti et al., Oncogene 2017; 36: 3505. (2) Lambert et al., Cell 2018; 175: 598. (3) Riggi et al., Cancer Cell 2014; 26: 668. (4) We thank Dr. Javed Khan and colleagues for pre-publication access to EWS tumor and cell line gene expression data.


















